.eslintignore

```diff
@@ -0,0 +1,3 @@
+coverage/
+node_modules/
+node-test-helpers/
```

Comment:
Not sure, was working on a few repos at the same time, so probably got confused, but I've updated it now.